.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "liveServer.settings.port": 5501
+  "liveServer.settings.port": 5502
 }
```

Comment:
- [ ] [OPTIONAL] Consider adding your IDE configuration to the `.gitignore` as this is not a part of the project